src/js/components/Beacon/BeaconSearchResults.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useAppSelector } from '@/hooks';
+import SearchResultsPane from '../Search/SearchResultsPane';
+
+const BeaconSearchResults = () => {
+  const isFetchingData = useAppSelector((state) => state.beaconQuery?.isFetchingQueryResponse);
```

Comment:
dont need `?` in the `beaconQuery?` here I think - it should always be an object (and if it isn't somehow, the line below can error)

Reply:
indeed yes